query-service-impl/src/main/java/org/hypertrace/core/query/service/QueryRequestUtil.java

```diff
@@ -68,4 +77,62 @@ public static Expression createNullNumberLiteralExpression() {
                 .setValue(Value.newBuilder().setValueType(ValueType.NULL_NUMBER)))
         .build();
   }
+
+  public static Filter createContainsKeyFilter(AttributeExpression complexAttributeExpression) {
+    return createContainsKeyFilter(
+        complexAttributeExpression.getAttributeId(), complexAttributeExpression.getSubpath());
+  }
+
+  public static Filter createContainsKeyFilter(String column, String value) {
+    return Filter.newBuilder()
+        .setOperator(Operator.CONTAINS_KEY)
+        .setLhs(createColumnExpression(column))
+        .setRhs(createStringArrayLiteralValueExpression(List.of(value)))
```

Comment:
Do we need an array as rhs?

Reply:
Reusing the same function used to create `CONTAINS_KEYVALUE` filter.

Reply:
As of today, yes - that's the improvement I referenced in the middle paragraph in https://github.com/hypertrace/query-service/pull/107#discussion_r765960111

Reply:
Ref - https://github.com/hypertrace/query-service/blob/696bacc67ebe2fbffe2605279b9aa47d2ed123b6/query-service-impl/src/main/java/org/hypertrace/core/query/service/pinot/QueryRequestToPinotSQLConverter.java#L351

Reply:
We can replace ones we deprecate `CONTAINS_KEYVALUE`